### ISSUE TITLE
reftest failure in border reftests

### ISSUE THREAD

**Issue by jdm** [servo/servo#4827]
From http://build.servo.org/builders/mac2/builds/892/steps/shell_1/logs/stdio:
```
failures:
    border_black_groove.html != border_black_solid.html
    border_black_ridge.html != border_black_groove.html
    border_black_ridge.html != border_black_solid.html
    inset_blackborder.html != blackborder_ref.html
    outset_blackborder.html != blackborder_ref.html
```

**Comment by mbrubeck** [comment]
Caused by #4825.

**Comment by Adenilson** [comment]
The fix: https://github.com/servo/servo/pull/4828

**Comment by mbrubeck** [comment]
Fixed for now by backing out #4825.
